frontend/src/pages/ActivityReport/Pages/Review/Submitter/Draft.js

```diff
@@ -91,9 +96,21 @@ const Draft = ({
         <div className="margin-top-3">
           <ApproverStatusList approverStatus={approverStatusList} />
         </div>
-        <Button outline type="button" onClick={() => { onSaveForm(false); }}>Save Draft</Button>
+        <Button outline type="button" onClick={() => { onSaveForm(false); updateShowSavedDraft(true); }}>Save Draft</Button>
```

Comment:
could we move this ```updateShowSavedDraft``` to a new line?

Reply:
I have moved spaced this over multiple lines.